tests/test_fitness_model.py

```diff
@@ -0,0 +1,203 @@
+"""
+Tests for the `fitness_model` module.
```

Comment:
So great to see tests. Something we haven't been good at and I'm glad you're making a push to include them. If you could give me an overview at some point on how you approach this, it would be helpful. I guess I need to read into PyTest.